app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -183,17 +183,27 @@ function initializePodcastPlayback() {
     Array.prototype.forEach.call(records, function (record) {
       var episodeSlug = record.getAttribute('data-episode');
       var podcastSlug = record.getAttribute('data-podcast');
-      record.onclick = function () {
-        if (podcastBarAlreadyExistAndPlayingTargetEpisode(episodeSlug)) {
-          var audio = getById('audio');
-          if (audio) {
-            playPause(audio);
+
+      var togglePodcastState = function (e) {
+        e.preventDefault();
+        var isPressed = record.getAttribute('aria-pressed') === 'true';
+        var isValidKey = e.type === 'keydown' && [32, 13].includes(e.keyCode);
+        if (isValidKey || e.type === 'click') {
+          if (podcastBarAlreadyExistAndPlayingTargetEpisode(episodeSlug)) {
+            var audio = getById('audio');
+            record.setAttribute('aria-pressed', !isPressed);
+
+            if (audio) {
+              playPause(audio);
+            }
+          } else {
+            stopRotatingActivePodcastIfExist();
+            loadAndPlayNewPodcast(episodeSlug);
           }
-        } else {
-          stopRotatingActivePodcastIfExist();
-          loadAndPlayNewPodcast(episodeSlug);
         }
       };
+      record.addEventListener('click', togglePodcastState);
+      record.addEventListener('keydown', togglePodcastState);
```

Comment:
refactor (blocking) - we should remove this keydown event listener. It's actually causing a focus trap at the moment in the UI - if you tab to the button, you can never tab away from it 🙃 

Since a button's 'click' event is triggered by either keyboard or mouse activation, there's no need for a separate keydown listener.

This will mean you can also remove line 190 and the check on line 191 👍